package.json

```diff
@@ -58,6 +60,9 @@
   "jest": {
     "preset": "ts-jest",
     "testEnvironment": "node",
-    "testTimeout": 20000
+    "testTimeout": 20000,
+    "moduleNameMapper": {
+      "\\.(css|scss)$": "<rootDir>/client/specs/__mocks__/styleMock.ts"
+    }
```

Comment:
I don't feel like we need to mock the styles 🤔 wdyt @dalejo96

Reply:
I did this because I was having trouble running client tests without mocking styles modules.